frontend/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -207,9 +212,10 @@ function* loadIsosurfaceWithNeighbors(
     seedPosition != null ? seedPosition : yield* select(state => getFlooredPosition(state.flycam));
   const clippedPosition = clipPositionToCubeBoundary(position, zoomStep, resolutionInfo);
   let positionsToRequest = [clippedPosition];
-  if (seedPosition) {
```

Comment:
@grittaweisheit do you know why `addIsosurfaceAction` is only dispatched if there is a `seedPosition` ? 1) The used position is set to the current annotation position and used as a seed if `seedPosition == null`, so it should be fine to use that here, too? 2) I'm not really sure in which cases `seedPosition` is null anyway. The signature states `seedPosition: ?Vector`, but all callers seem to pass it if I'm not missing anything?

Reply:
I think it is because `ensureSuitableIsosurface` is also called at every flycam movement, but we don't really want to add an isosurface then. And since there is no seed position in that case, that is the way I decided whether to dispatch the `addIsosurfaceAction` or not.

Reply:
Ok, makes sense. Good point with the flycam-movement listener! I missed that. However, the "if seed position is null, we know only it was caused by a flycam movement"-logic is a bit too hidden for me. I just changed the code to simply dispatch `addIsosurfaceAction` everytime **unless** it already exists in the store. This is bit more intuitive and safer in my opinion.